sample/src/main/kotlin/com/slack/circuit/sample/home/HomePresenter.kt

```diff
@@ -168,3 +253,106 @@ private fun BottomNavigationBar(selectedIndex: Int, onSelectedIndex: (Int) -> Un
     }
   }
 }
+
+@Composable
+private fun GenderFilterOption(
+  state: PetListFilterScreen.State,
+  eventSink: (HomeScreen.Event) -> Unit
+) {
+  Box { Text(text = "Gender") }
+  Row(modifier = Modifier.selectableGroup()) {
+    Column {
+      Text(text = "All")
+      RadioButton(
+        selected = state.gender == Gender.ALL,
+        onClick = {
+          eventSink(
+            HomeScreen.Event.PetListFilterEvent(
+              PetListFilterScreen.Event.FilterByGender(Gender.ALL)
+            )
+          )
+        }
+      )
+    }
+    Column {
+      Text(text = "Male")
+      RadioButton(
+        selected = state.gender == Gender.MALE,
+        onClick = {
+          eventSink(
+            HomeScreen.Event.PetListFilterEvent(
+              PetListFilterScreen.Event.FilterByGender(Gender.MALE)
+            )
+          )
+        }
+      )
+    }
+    Column {
+      Text(text = "Female")
+      RadioButton(
+        selected = state.gender == Gender.FEMALE,
+        onClick = {
+          eventSink(
+            HomeScreen.Event.PetListFilterEvent(
+              PetListFilterScreen.Event.FilterByGender(Gender.FEMALE)
+            )
+          )
+        }
+      )
+    }
+  }
+}
+
+@Composable
+private fun SizeFilterOption(
+  state: PetListFilterScreen.State,
+  eventSink: (HomeScreen.Event) -> Unit
+) {
+  Box { Text(text = "Size") }
+  Row(modifier = Modifier.selectableGroup()) {
+    Column {
+      Text(text = "All")
+      RadioButton(
+        selected = state.size == Size.ALL,
+        onClick = {
+          eventSink(
+            HomeScreen.Event.PetListFilterEvent(PetListFilterScreen.Event.FilterBySize(Size.ALL))
+          )
+        }
+      )
+    }
+    Column {
+      Text(text = "Small")
+      RadioButton(
+        selected = state.size == Size.SMALL,
+        onClick = {
+          eventSink(
+            HomeScreen.Event.PetListFilterEvent(PetListFilterScreen.Event.FilterBySize(Size.SMALL))
+          )
+        }
+      )
+    }
+    Column {
+      Text(text = "Medium")
+      RadioButton(
+        selected = state.size == Size.MEDIUM,
+        onClick = {
+          eventSink(
+            HomeScreen.Event.PetListFilterEvent(PetListFilterScreen.Event.FilterBySize(Size.MEDIUM))
+          )
+        }
+      )
+    }
+    Column {
+      Text(text = "Large")
```

Comment:
nit: seems like we should store this string in the enum and then we could iterate over the enum values to populate these? Same with the gender selection

Reply:
Good point, addressed here https://github.com/slackhq/circuit/pull/101/commits/89de7f6d29b498d957c17c4ec559be364ecd09c1